.pre-commit-hooks.yaml

```diff
@@ -0,0 +1,6 @@
+-   id: swiftlint
```

Comment:
Minor nit: we indent `.swiftlint.yml` with two spaces, can you do the same here?

Reply:
Done